corehq/apps/app_manager/app_translations/generators.py

```diff
@@ -174,8 +174,10 @@ def _filter_invalid_rows_for_form(self, rows, form_id, label_index):
         valid_rows = []
         for i, row in enumerate(rows):
             question_label = row[label_index]
-            if not any('SKIP TRANSIFEX' in comment for comment in form_labels_to_comments[question_label]):
-                valid_rows = rows
+            if question_label in form_labels_to_comments:
+                if any('SKIP TRANSIFEX' in comment for comment in form_labels_to_comments[question_label]):
+                    continue
+            valid_rows.append(row)
```

Comment:
Was this old format just confused?

Reply:
This logic could be simplified a bit if you pushed the `'SKIP TRANSIFEX'` check into `_get_labels_to_comments` - that is, make that method be responsible for determining whether to send to transifex.  It could return a boolean for each label, rather than a list of comments (since the comments aren't needed beyond that check):
```python
self._skip_questions[form_id][question_label]
```

Reply:
> Was this old format just confused?

yes.

Reply:
Thanks @esoergel . Updated to filter out earlier instead.